test/metabase/server/middleware/security_test.clj

```diff
@@ -5,7 +5,11 @@
    [clojure.string :as str]
    [clojure.test :refer :all]
    [metabase.config :as config]
-   [metabase.server.middleware.security :as mw.security]
+   [metabase.server.middleware.security :as mw.security :refer [approved-domain?
+                                                                approved-origin?
+                                                                approved-port?
+                                                                approved-protocol?
+                                                                parse-url]]
```

Comment:
I won't block the review for this, but in my opinion it's better to use the usual namespace prefix when testing, that way you get a feel for how using it will be. 

For example with a namespace like `metabase.user.email`, and a function is called `send-user-email!`, well `user.email/send-user-email!` makes it easier to notice that the function's name is redundant, and could just be called `send!` ->  `user.email/send!`.

Reply:
Thanks, I changed it :)